torch/signal/windows/windows.py

```diff
@@ -0,0 +1,297 @@
+import torch
+from math import sqrt
+
+from torch import Tensor
+from torch._prims_common import is_float_dtype
+from torch._torch_docs import factory_common_args, parse_kwargs, merge_dicts
+
+__all__ = [
+    'cosine',
+    'exponential',
+    'gaussian',
+]
+
+window_common_args = merge_dicts(
+    parse_kwargs(
+        """
+    M (int): the length of the window.
+        In other words, the number of points of the exponential window.
```

Comment:
This will be written for windows like the cosine window below, but it says "number of points of the exponential window"?

Reply:
Right! Good catch, let me fix that.